openwisp_users/tests/test_admin.py

```diff
@@ -1353,7 +1353,7 @@ def test_delete_org_user(self):
             )
             self.assertContains(r, msg)
             post_data.update({'post': 'yes'})
-            with self.assertNumQueries(17):
+            with self.assertNumQueries(21):
```

Comment:
can you add a comment before each assertNumQueries like:

`# django-reversion adds ~4 queries`

Reply:
Done :+1: